src/python/tests/core/bot/tasks/corpus_pruning_task_test.py

```diff
@@ -421,6 +420,10 @@ def tearDown(self):
   def test_prune(self):
     """Test pruning."""
     self._setup_env(job_type='libfuzzer_asan_job')
+    self.mock._record_cross_pollination_stats.side_effect = (
+        self.get_mock_record_compare('test_fuzzer', 'random', 'test2_fuzzer',
```

Comment:
nit: IMO it would be more readable if you created an object here, like the one in `src/python/bot/untrusted_runner/tasks_impl.py` line 56 and then asserted its fields

Reply:
Discussed on chat: Used named arguments to make this more readable.

Reply:
or just use named arguments